lib/rubocop/cop/layout/class_structure.rb

```diff
@@ -234,10 +239,11 @@ def class_elements(class_node)
           end
         end
 
-        def ignore?(classification)
+        def ignore?(classification, node)
           classification.nil? ||
             classification.to_s.end_with?('=') ||
-            expected_order.index(classification).nil?
+            expected_order.index(classification).nil? ||
+            classification == 'constants' && dynamic_constant?(node)
```

Comment:
Is `classification == 'constants'` condition necessary? I think that this node type can be specified by `dynamic_constant?`.

```
def_node_matcher :dynamic_constant?, <<~PATTERN
  (casgn nil? _ (send ...))
   ^^^^^
```

Reply:
Right, `casgn` takes care of that.